pkg/oauthserver/auth.go

```diff
@@ -292,7 +292,7 @@ func (c *OAuthServerConfig) getAuthenticationHandler(mux oauthserver.Mux, errorH
 	multiplePasswordProviders := false
 	passwordProviderCount := 0
 	for _, identityProvider := range c.ExtraOAuthConfig.Options.IdentityProviders {
-		if config.IsPasswordAuthenticator(identityProvider) && identityProvider.UseAsLogin {
+		if (config.IsPasswordAuthenticator(identityProvider) || config.IsBootstrap(identityProvider)) && identityProvider.UseAsLogin {
```

Comment:
no need to change this line

Reply:
fixed, originally i took the bootstrap out of the IsPassword cases :facepalm: thanks